cpp/inc/bond/protocol/simple_binary_impl.h

```diff
@@ -6,6 +6,7 @@
 #include <bond/core/config.h>
 
 #include "simple_binary.h"
+#include <bond/core/detail/protocol_visitors.h>
```

Comment:
Minor: blank line between bond and !bond headers, please.

Reply:
These are both bond headers, do you still want me to separate them?

Reply:
Separated as they're coming from different bond modules

Reply:
Yes, please. Thanks!